src/main/java/sleepy/tasks/Deadline.java

```diff
@@ -0,0 +1,45 @@
+package sleepy.tasks;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * This class is a type of task.
+ *
+ * @author kjw142857
+ */
+public class Deadline extends Task {
```

Comment:
Would renaming to DeadlineTask be better? To me Deadline alone does not give the sense of it being a task.

Reply:
Hmm, I think you make a very good point here, will consider your suggestion!